Explorer/Assets/Scripts/ECS/StreamableLoading/GLTF/GltFastDownloadProvider.cs

```diff
@@ -56,24 +63,40 @@ public async Task<IDownload> RequestAsync(Uri uri)
             }
 
             uri = new Uri(tryGetContentUrlResult);
+            var commonArguments = new CommonArguments(URLAddress.FromString(GetUrl(uri)));
+
+            byte[] data = Array.Empty<byte>();
+            string error = string.Empty;
+            string text = string.Empty;
+            bool success;
+
+            try
+            {
+                var downloadHandler = await webRequestController.GetAsync(commonArguments, new CancellationToken(), reportData).ExposeDownloadHandlerAsync();
```

Comment:
You need to dispose of `downloadhandler` manually (in `finally`) as it is exposed and is not disposed of automatically anymore